src/integration/java/io/pinecone/integration/dataPlane/QueryErrorTest.java

```diff
@@ -25,15 +26,16 @@ public class QueryErrorTest {
     @BeforeAll
     public static void setUp() throws IOException, InterruptedException {
         PineconeConnection connectionMock = mock(PineconeConnection.class);
```

Comment:
This might be a dumb question, but shouldn't the `pineconeMock` be able to be created from the `connectionMock` obj?

Reply:
the constructor for Pinecone accepts PineconeConfig config and ManageIndexesApi manageIndexesApi, so we can't instantiate it using PineconeConnection class